### PR TITLE
fix(activity): show indexing wallet status

### PR DESCRIPTION
**Pull request by th0rOdinson**

### CODE REVIEW

File: apps/root/src/common/utils/transaction-history/index.ts [hunk 1]
```diff
@@ -186,3 +186,34 @@ export const getTransactionPriceColor = (tx: TransactionEvent) => {
...
+  switch (tx.type) {
+    case TransactionEventTypes.ERC20_APPROVAL:
+      const { owner, spender } = tx.data;
+      wallets = [owner, spender];
+      break;
+    case TransactionEventTypes.DCA_WITHDRAW:
+    case TransactionEventTypes.DCA_TERMINATED:
+    case TransactionEventTypes.DCA_MODIFIED:
+    case TransactionEventTypes.DCA_PERMISSIONS_MODIFIED:
+    case TransactionEventTypes.DCA_CREATED:
```
Comment (FiboApe) [inline apps/root/src/common/utils/transaction-history/index.ts:201]: DCA_CREATED could have a different user to the one that initiated the tx

Comment (FiboApe) [reply apps/root/src/common/utils/transaction-history/index.ts:201]: Not through our UI, but it is available through SC so an event could come with those two values different

File: apps/root/src/hooks/useIsSomeWalletIndexed.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const INDEXER_ACCEPTANCE = 0.95; // 95%
+
+export default function useIsSomeWalletIndexed(wallet?: Address) {
+  const { history } = useStoredTransactionHistory();
+
+  const isSomeWalletIndexed = React.useMemo(() => {
+    const returnValue = Object.entries(history?.indexing || {}).some(([address, chainsData]) => {
+      if (!wallet || wallet === address) {
+        const isSomeChainIndexed = Object.values(chainsData).some((chainData) => {
```
Comment (FiboApe) [inline apps/root/src/hooks/useIsSomeWalletIndexed.ts:13]: you can just return this

File: apps/root/src/hooks/useIsSomeWalletIndexed.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { Address } from 'viem';
+
+const INDEXER_ACCEPTANCE = 0.95; // 95%
+
+export default function useIsSomeWalletIndexed(wallet?: Address) {
+  const { history } = useStoredTransactionHistory();
+
+  const isSomeWalletIndexed = React.useMemo(() => {
+    const returnValue = Object.entries(history?.indexing || {}).some(([address, chainsData]) => {
+      if (!wallet || wallet === address) {
```
Comment (FiboApe) [inline apps/root/src/hooks/useIsSomeWalletIndexed.ts:12]: should return false if this is false right?

File: apps/root/src/hooks/useIsSomeWalletIndexed.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import useStoredTransactionHistory from './useStoredTransactionHistory';
+import { Address } from 'viem';
+
+const INDEXER_ACCEPTANCE = 0.95; // 95%
+
+export default function useIsSomeWalletIndexed(wallet?: Address) {
+  const { history } = useStoredTransactionHistory();
+
+  const isSomeWalletIndexed = React.useMemo(() => {
+    const returnValue = Object.entries(history?.indexing || {}).some(([address, chainsData]) => {
```
Comment (FiboApe) [inline apps/root/src/hooks/useIsSomeWalletIndexed.ts:11]: Also here, you could return this directlty

File: apps/root/src/common/utils/transaction-history/index.ts (hunk 1 above)
Comment (th0rOdinson) [reply apps/root/src/common/utils/transaction-history/index.ts:201]: Pretty cool!

**Review by FiboApe — APPROVED** [review]

**Pull request merged**